### PR TITLE
added start dm screen

### PR DESCRIPTION
**Pull request by bountyforager**
Added Start Dm screen

### CODE REVIEW

File: lib/ui/view/start_dm/start_dm_view.dart [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                      Row(
+                        children: [
+                          Expanded(
+                            child: Container(
+                              height: 56,
+                              margin: EdgeInsets.only(left: 13.0),
+                              alignment: Alignment.centerLeft,
+                              child: FocusScope(
+                                child: Focus(
+                                  onFocusChange: (focus) {
```
Comment (Quwaysim) [inline lib/ui/view/start_dm/start_dm_view.dart:70]: logic can be moved to viewmodel

File: lib/ui/view/start_dm/start_dm_viewmodel.dart [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
```
Comment (Quwaysim) [inline lib/ui/view/start_dm/start_dm_viewmodel.dart:1]: remove material import, use [FormView](https://www.filledstacks.com/post/building-flutter-login-and-sign-up-forms/) to use TEController without importing material

File: lib/ui/view/start_dm/widgets/custom_chip_input.dart [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  final Future<List<UserModel>> mockResults;
+  final horizontalSpace = SizedBox(width: 12);
+ 
+
+  @override
+  Widget build(BuildContext context) {
+    return ChipsInput(
+      key: _chipKey,
+      textCapitalization: TextCapitalization.words,
+      textStyle: const TextStyle(
```
Comment (Quwaysim) [inline lib/ui/view/start_dm/widgets/custom_chip_input.dart:27]: use one of the styles in styles.dart or add yours if it's not there

**Review by Quwaysim — CHANGES_REQUESTED** [review]

File: lib/ui/view/start_dm/start_dm_viewmodel.dart (hunk 2 above)
Comment (bountyforager) [reply lib/ui/view/start_dm/start_dm_viewmodel.dart:1]: I don't understand this, and seeing as there is only 15 minutes. Please accept this. I will change it later

**Pull request closed without merging**